src/board/system76/common/kbled/rgb_i2c.c

```diff
@@ -39,10 +39,31 @@ uint8_t kbled_get(void) {
     return level;
 }
 
+uint8_t kbled_max(void) {
+    return 255;
+}
+
 void kbled_set(uint8_t level) {
     kbled_i2c_set(0x12, &level, 1);
 }
 
+uint32_t kbled_get_color(void) {
+    // Get blue component from channel 0
+    uint8_t value;
+    kbled_i2c_get(0x02, &value, 1);
+    uint32_t color = (uint32_t)value;
+
+    // Get green component from channel 3
+    kbled_i2c_get(0x05, &value, 1);
```

Comment:
Same issue as set_color for darp5/oryp5: addresses for color components are different.